### PR TITLE
Fix concept import in the single csv importer, #9078

### PR DESCRIPTION
**Pull request by njkim**
### Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Add the config to properly transform data for concept datatype, #9078 
Also check if the value is empty string before try to transform, if the value is "" turn it into None.
Fix the issue the single csv importer for the concept and resource instance datatype

### Issues Solved

#9078 

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

File: arches/app/etl_modules/import_single_csv.py [hunk 1]
```diff
@@ -280,12 +280,22 @@ def populate_staging_table(self, loadid, graphid, has_headers, fieldnames, csv_f
                                 config = current_node.config
                                 config["path"] = temp_dir
                                 value = (
-                                    datatype_instance.transform_value_for_tile(source_value, **config) if source_value is not None else None
+                                    datatype_instance.transform_value_for_tile(source_value, **config)
+                                    if source_value is not None and source_value != ""
+                                    else None
```
Comment (chrabyrd) [inline arches/app/etl_modules/import_single_csv.py:285]: is it possible to change this to:

value = datatype_instance.transform_value_for_tile(source_value, **config) if source_value else None

? Or is there an explicit reason for the `None` and empty string comparison?

File: arches/app/etl_modules/import_single_csv.py [hunk 2]
```diff
@@ -280,12 +280,22 @@ def populate_staging_table(self, loadid, graphid, has_headers, fieldnames, csv_f
...
                                 )
                                 errors = datatype_instance.validate(value, nodeid=node, path=temp_dir)
                             else:
-                                value = datatype_instance.transform_value_for_tile(source_value) if source_value is not None else None
-                                errors = datatype_instance.validate(value)
+                                try:
+                                    config = current_node.config
+                                except TypeError:
+                                    config = {}
```
Comment (chrabyrd) [inline arches/app/etl_modules/import_single_csv.py:292]: why the try/except here? is it possible to check for the current_node config a different way?

File: arches/app/etl_modules/import_single_csv.py [hunk 3]
```diff
@@ -280,12 +280,22 @@ def populate_staging_table(self, loadid, graphid, has_headers, fieldnames, csv_f
...
+                                value = (
+                                    datatype_instance.transform_value_for_tile(source_value, **config)
+                                    if source_value is not None and source_value != ""
+                                    else None
+                                )
```
Comment (chrabyrd) [inline arches/app/etl_modules/import_single_csv.py:297]: same note here as the previous value assignment

**Review by chrabyrd — CHANGES_REQUESTED** [review]
looks good, just a few questions about format

File: arches/app/etl_modules/import_single_csv.py (hunk 1 above)
Comment (njkim) [reply arches/app/etl_modules/import_single_csv.py]: good point! thanks.

File: arches/app/etl_modules/import_single_csv.py (hunk 2 above)
Comment (njkim) [reply arches/app/etl_modules/import_single_csv.py]: Yes you are right. There should always be the config. So, remove try and move the line to above combine with file type. Thanks.

**Review by chrabyrd — APPROVED** [review]
lgtm!

**Comment by chrabyrd** [comment]
resolved in #9079

**Pull request merged**